mappings/witch_5.0.yaml

```
@@ -0,0 +1,45 @@
+model: WITCH 5.0
+native_regions:
+  - BRAZIL: WITCH 5.0|BRAZIL
```

Please use not all-capitalized in sync with the definitions file, i.e.
```suggestion
  - BRAZIL: WITCH 5.0|Brazil
```